pkgs/tools/misc/coreutils/default.nix

```diff
@@ -76,8 +76,7 @@ stdenv.mkDerivation (rec {
   outputs = [ "out" "info" ];
 
   nativeBuildInputs = [ perl xz.bin ]
-    ++ optionals stdenv.hostPlatform.isCygwin [ autoreconfHook texinfo ]   # due to patch
-    ++ optionals stdenv.hostPlatform.isMusl [ autoreconfHook bison ];   # due to patch
+    ++ optionals stdenv.hostPlatform.isCygwin [ autoreconfHook texinfo ];  # due to patch
```

Comment:
This is already done in your other PR. Added by mistake?

Reply:
No, I plan to merge this after the other has gone in (I believe that this depends on the other to even get far enough)

Reply:
I didn't see that it is in its own commit. It should work then.